.gitignore

```diff
@@ -31,4 +31,8 @@ libInertialSenseSDK.a
 scripts/.venv/
 python/venv/
 python/logInspector/.eggs/
-
+.cmake/
+/install/
+/log/
+/ros2/install/
+/ros2/log/
```

Comment:
Should also add .ninja to the .gitnore, and while we're at it, remove the two .ninja files below.  They do not belong in the PR/repo.